apps/hip/templates/includes/sidebar.html

```diff
@@ -44,9 +44,18 @@ <h5 class="is-size-7 desktop-hidden-hip">Response Partners</h5>
     <div class="ml-2 mr-6 sidebar-divider-hip desktop-hidden-hip"></div>
   </div>
   <ul class="mb-4 desktop-hidden-hip {% if hidden_desktop %}is-hidden-desktop{% endif %}">
-    <li class="pl-4 pb-2 sidenav-link-hip" data-href="#closed-pod-login">
-      <span class="signin-icon-blue-hip"></span>
-      Partner Log In
+    <li class="pl-4 pb-2 sidenav-link-hip" data-href="#closed-pod-login-logout">
+      {% if request.user.is_authenticated %}
+        <a href="{% url 'logout' %}" class="is-block alert-hip">
+          <span class="signin-icon-red-hip"></span>
+            Logout
```

Comment:
```suggestion
            Partner Log out
```
I think we should check with Erik and Kat, but I think this text is what the wireframes has.

Reply:
I based it on https://www.figma.com/file/OZAfVKxh8fs8XE7RLnlbY7/PDPH-COVID19?node-id=1394%3A34615. I'll check with Erik about the correct words

Reply:
Yes, thanks @dchukhin

Reply:
🤝